components/nds-core/scss/helpers/_helpers-print.scss

```diff
@@ -0,0 +1,42 @@
+////
+/// @group helpers
+////
+
+/// Styles to hide and show under the print media query
+
+/// Hide the element when printing
+/// @output the CSS to hide on the print stylesheet
+/// @since 1.0.0
+@mixin nds-hide-print {
+  @media print {
+    display: none !important;
+  }
+}
+
+///Only show the element when printing (not for other media)
+/// @output the CSS to hide on all media except print
+/// @since 1.0.0
+@mixin nds-show-print {
+  display: none !important;
+  @media print {
```

Comment:
If we're going to keep @mixin nds-print then we should use it here

Reply:
Also, we should then be consistent with all the other print media queries e.g. https://github.com/nice-digital/nice-design-system/blob/v1/components/design-system/scss/typography/_typography-quotations.scss#L29-L31